src/aead/gcm.rs

```diff
@@ -65,7 +65,7 @@ impl Key {
                 }
             }
 
-            #[cfg(all(not(target_arch = "aarch64"), not(target_env = "sgx")))]
+            #[cfg(all(not(target_arch = "aarch64")))]
```

Comment:
https://github.com/crypto-com/ring/pull/5#pullrequestreview-543400995 -- no need for "all" + why not to use the HW implementation?

Reply:
> no need for "all" 
Will change this.

> HW implementation
I'm probably missing something, cause I'm new to the sgx stuff, but I thought we were trying to avoid the changes from this patch https://github.com/briansmith/ring/pull/738/commits/7c29308410aa4cb06874fdac3caaff1802073860 and the `panic!` state